xds/src/main/java/io/grpc/xds/internal/rbac/engine/GrpcAuthorizationEngine.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright 2021 The gRPC Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.grpc.xds.internal.rbac.engine;
+
+import io.envoyproxy.envoy.config.rbac.v3.Policy;
+import io.envoyproxy.envoy.config.rbac.v3.RBAC;
+import io.envoyproxy.envoy.config.rbac.v3.RBAC.Action;
+import io.grpc.xds.EvaluateArgs;
+import io.grpc.xds.internal.rbac.engine.AuthorizationEngineInterface.AuthDecision.DecisionType;
+import java.util.HashMap;
+import java.util.Map;
+
+/** Implementation of gRPC server access control based on envoy RBAC protocol:
```

Comment:
nit: we follow the convention of starting the Javadoc from a the second line of block comment.

Reply:
fixed.